controllers/masteruserrecord/sync.go

```diff
@@ -183,32 +183,41 @@ func (s *Synchronizer) alignReadiness() (bool, error) {
 	}
 
 	if condition.IsNotTrue(s.record.Status.Conditions, toolchainv1alpha1.MasterUserRecordUserProvisionedNotificationCreated) {
-		notification := &toolchainv1alpha1.Notification{
-			ObjectMeta: v1.ObjectMeta{
-				GenerateName: fmt.Sprintf("%s-%s-", s.record.Name, toolchainv1alpha1.NotificationTypeProvisioned),
-				Namespace:    s.record.Namespace,
-				Labels: map[string]string{
-					// NotificationUserNameLabelKey is only used for easy lookup for debugging and e2e tests
-					toolchainv1alpha1.NotificationUserNameLabelKey: s.record.Name,
-					// NotificationTypeLabelKey is only used for easy lookup for debugging and e2e tests
-					toolchainv1alpha1.NotificationTypeLabelKey: toolchainv1alpha1.NotificationTypeProvisioned,
-				},
-			},
-			Spec: toolchainv1alpha1.NotificationSpec{
-				// The UserID property actually refers to the UserSignup resource name.  This will be renamed
-				// (or removed) in a future PR
-				UserID:   s.record.Labels[toolchainv1alpha1.MasterUserRecordOwnerLabelKey],
-				Template: notificationtemplates.UserProvisioned.Name,
-			},
-		}
-
-		err := controllerutil.SetControllerReference(s.record, notification, s.scheme)
-		if err != nil {
+		labels := map[string]string{
+			// NotificationUserNameLabelKey is only used for easy lookup for debugging and e2e tests
```

Comment:
It is not for tests or debugging anymore. You do rely on this label in the controller now.

Reply:
right :+1: 
I just blindly copy-pasted it and didn't pay attention on the comments - fixed in ea4f95c